spk/firebird/src/service-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+FBGUARD_BIN_FILE="${SYNOPKG_PKGDEST}/bin/fbguard"
+
+# We set onetime because we want the systemd to take care about the failures
+SERVICE_COMMAND="${FBGUARD_BIN_FILE} -pidfile ${PID_FILE} -daemon -onetime"
+
+SYSDBA_PASSWORD_FILE="${SYNOPKG_PKGDEST}/SYSDBA.password"
+
+service_postinst () {
+    set -x
+    cd "$SYNOPKG_PKGDEST" || exit 1
```

Comment:
this could never happen.

Why not declaring
`PATH=${SYNOPKG_PKGDEST}/bin:${PATH}`
at the top of this file?

Reply:
Most of the scripts "just works" when executed from the package root. Running them from other location might require more fiddling with the compilation parameters. I can try, but I would prefer not to.